Cargo.toml

```diff
@@ -98,14 +102,15 @@ rpassword = "7.3.1"
 futures = "0.3.30"
 bytes = "1.5.0"
 which = "6.0.0"
+secrecy = "0.8.0"
 wit-bindgen-rust = "0.21.0"
 wit-bindgen-core = "0.21.0"
 tempfile = "3.10.1"
 assert_cmd = "2.0.14"
 predicates = "3.1.0"
 wasmparser = "0.201.0"
 wat = "1.201.0"
-warg-server = "0.3.0"
+warg-server = "0.4.1"
```

Comment:
Nit (and not your issue): do you mind moving this up with the other warg-related deps? It's only a dev dependency, but cargo doesn't distinguish between dev and non-dev deps at the workspace level. Thanks!